BridgeCareApp/VuejsApp/src/shared/AppModalPopup.vue

```diff
@@ -0,0 +1,48 @@
+﻿<template>
```

Comment:
This modal is specific for a warning message. I think this can be made more generic so that it can become a reusable component throughout (think 'alert' modal).